src/MEDS_DEV/helpers/generate_predictions.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+export MEDS_ROOT_DIR=$1
+export MEDS_DATASET_NAME=$2
+export MEDS_TASK_NAME=$3
+
+shift 3
+
+MEDS_DEV_REPO_DIR=$(python -c "from importlib.resources import files; print(files(\"MEDS_DEV\"))")
+export MEDS_DEV_REPO_DIR
+
+# TODO improve efficiency of prediction generator by using this
```

Comment:
What is this TODO for? I'm not sure I understand.

Reply:
At the moment I'm reimplementing recursive search through the directory to find shards for which to generate the predictions, but I think this could be improved with the `expand_shards` helper you have implemented.